guides/distributed-ets.md

```diff
@@ -0,0 +1,86 @@
+# Distributed Rate Limiter with ETS Backend
+
+The example implements a distributed, eventually consistent rate limiter using Phoenix.PubSub for broadcasting each hit across nodes and a local ETS backend to manage rate-limiting counters. This setup is useful when you need to limit the number of actions (e.g., requests) across multiple nodes in a cluster.
```

Comment:
why not make it backend that is usable then by default in a separate MR?

Reply:
I think there too many unknowns to make it a backend. There might be different ways people might want to use it or have different assumptions about how it works, and this way it's clear because the code is shown up front and it's up to them to adapt it to their needs.

Reply:
fair. we can make it later but it would still be nice to have a default opinion that works so people can use it and then they can copy it or we can make it more configurable over time. Fine for now to document it. I can add i later the way I want it by default